src/js/tabs/advanced.controller.js

```diff
@@ -67,6 +67,7 @@ AdvancedTab.prototype.angular = function(module)
           // This has to be updated manually because the network object is not
           // recreated unless we do location.reload()
           network.remote.max_fee = $scope.options.max_tx_network_fee;
+          $scope.userBlob.set('/clients/rippletradecom/maxNetworkFee', $scope.options.max_tx_network_fee)
```

Comment:
Ok just checking. Wasn't sure if we were changing blobvault with our identity program.